lxc/config/remote.go

```diff
@@ -78,12 +78,20 @@ func (c *Config) GetInstanceServer(name string) (lxd.InstanceServer, error) {
 		d, err := lxd.ConnectLXDUnix(strings.TrimPrefix(strings.TrimPrefix(remote.Addr, "unix:"), "//"), args)
 		if err != nil {
 			var netErr *net.OpError
+			socketPath := ""
 
 			if errors.As(err, &netErr) {
-				return nil, fmt.Errorf("The LXD daemon doesn't appear to be started (socket path: %s)", netErr.Addr)
+				socketPath = fmt.Sprintf(" (socket path: %s)", netErr.Addr)
 			}
 
-			return nil, err
+			switch {
+			case errors.Is(err, os.ErrNotExist):
+				return nil, fmt.Errorf("The LXD daemon doesn't appear to be started%s", socketPath)
```

Comment:
Suggest doing something a little simpler and more informative here:

```go
return nil, fmt.Errorf("LXD unix socket file %q not found. Please check LXD is running", netErr.Addr)
```

and 

```go
return nil, fmt.Errorf("LXD unix socket file %q not accessible . Please check the file permissions", netErr.Addr)
```

Reply:
I was shooting to reduce the line length a little, but I agree that this is a marked improvement.